### ISSUE TITLE
Crash on boot

### ISSUE THREAD

**Issue by brauliobo** [ForestAdmin/forest-rails#215]
If I change `ActiveRecord::Base.connection.instance_values["config"][:adapter]` to `ActiveRecord::Base.connection.config[:adapter]` it works

```
/home/braulio/.rvm/gems/ruby-2.4.2/gems/forest_liana-2.0.3/lib/forest_liana/bootstraper.rb:174:in `send_apimap': undefined method `[]' for nil:NilClass (NoMethodError)
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/forest_liana-2.0.3/lib/forest_liana/bootstraper.rb:29:in `perform'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/forest_liana-2.0.3/lib/forest_liana/engine.rb:51:in `block in <class:Engine>'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/activesupport-4.2.9/lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/activesupport-4.2.9/lib/active_support/lazy_load_hooks.rb:45:in `block in run_load_hooks'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/activesupport-4.2.9/lib/active_support/lazy_load_hooks.rb:44:in `each'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/activesupport-4.2.9/lib/active_support/lazy_load_hooks.rb:44:in `run_load_hooks'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/application/finisher.rb:62:in `block in <module:Finisher>'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/initializable.rb:30:in `instance_exec'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/initializable.rb:30:in `run'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/initializable.rb:55:in `block in run_initializers'
        from /home/braulio/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/tsort.rb:228:in `block in tsort_each'
        from /home/braulio/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
        from /home/braulio/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/tsort.rb:431:in `each_strongly_connected_component_from'
        from /home/braulio/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/tsort.rb:349:in `block in each_strongly_connected_component'
        from /home/braulio/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/tsort.rb:347:in `each'
        from /home/braulio/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/tsort.rb:347:in `call'
        from /home/braulio/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/tsort.rb:347:in `each_strongly_connected_component'
        from /home/braulio/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/tsort.rb:226:in `tsort_each'
        from /home/braulio/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/tsort.rb:205:in `tsort_each'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/initializable.rb:54:in `run_initializers'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/application.rb:352:in `initialize!'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/railtie.rb:194:in `public_send'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/railtie.rb:194:in `method_missing'
        from /home/braulio/Projects/olery/oat/spec/dummy/config/environment.rb:5:in `<top (required)>'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/activesupport-4.2.9/lib/active_support/dependencies.rb:274:in `require'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/activesupport-4.2.9/lib/active_support/dependencies.rb:274:in `block in require'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/activesupport-4.2.9/lib/active_support/dependencies.rb:240:in `load_dependency'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/activesupport-4.2.9/lib/active_support/dependencies.rb:274:in `require'
        from /home/braulio/Projects/olery/oat/spec/dummy/config.ru:3:in `block in <main>'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/rack-1.6.8/lib/rack/builder.rb:55:in `instance_eval'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/rack-1.6.8/lib/rack/builder.rb:55:in `initialize'
        from /home/braulio/Projects/olery/oat/spec/dummy/config.ru:in `new'
        from /home/braulio/Projects/olery/oat/spec/dummy/config.ru:in `<main>'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/rack-1.6.8/lib/rack/builder.rb:49:in `eval'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/rack-1.6.8/lib/rack/builder.rb:49:in `new_from_string'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/rack-1.6.8/lib/rack/builder.rb:40:in `parse_file'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/rack-1.6.8/lib/rack/server.rb:300:in `build_app_and_options_from_config'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/rack-1.6.8/lib/rack/server.rb:209:in `app'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/commands/server.rb:61:in `app'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/rack-1.6.8/lib/rack/server.rb:337:in `wrapped_app'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/commands/server.rb:139:in `log_to_stdout'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/commands/server.rb:78:in `start'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/commands/commands_tasks.rb:80:in `block in server'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/commands/commands_tasks.rb:75:in `tap'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/commands/commands_tasks.rb:75:in `server'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
        from /home/braulio/.rvm/gems/ruby-2.4.2/gems/railties-4.2.9/lib/rails/commands.rb:17:in `<top (required)>'
        from script/rails:6:in `require'
        from script/rails:6:in `<main>'
```

**Comment by arnaudbesnier** [comment]
Hi @brauliobo, thanks for reporting this.
I understand you're using Ruby 2.4.2. What is your Rails version? It would help us a lot the reproduce, understand and fix your issue.

**Comment by brauliobo** [comment]
hi @arnaudbesnier, it is rails 4.2.9

**Comment by brauliobo** [comment]
I'm also using octopus @arnaudbesnier

**Comment by brauliobo** [comment]
@arnaudbesnier I'm waiting on this to move to production

**Comment by brauliobo** [comment]
any update on this?

**Comment by arnaudbesnier** [comment]
Hi @brauliobo, sorry for the very very late response.
I just released a patch that prevents this crash on server start.
It is available on the liana version 2.2.1.

🌲🌲🌲
